main.rb

```diff
@@ -0,0 +1,93 @@
+require_relative 'planet.rb'
+require_relative 'solar_system.rb'
+require 'pry'
+
+def main
+
+  star = SolarSystem.new('Sol')
+
+  earth = Planet.new('Earth', 'blue-green', 5.972e24, 1.496e8, 'Only planet known to support life')
+
+  jupiter = Planet.new('Jupiter', 'white-red-orange-brown', 1.898e27, 483.8, 'Largest planet in the Solar System and is 318 times as massive as Earth')
+
+  mars = Planet.new('Mars', 'red', 6.39e23, 141.6, 'The diameter of Mars is 4,212 miles')
+
+  star.add_planet(earth)
+  star.add_planet(jupiter)
+  star.add_planet(mars)
+
+  # verifies user answer for the intro of the program
+  def verify_user_answer(user_answer)
+    until user_answer == "1" || user_answer == "2" || user_answer == "3"
+      puts "That's not a valid answer. Please enter 1, 2, or 3"
```

Comment:
I like that you've split these bits of functionality out into separate methods, but for organization they should probably be defined outside of the `main` method.